src/appleseed/foundation/image/genericprogressiveimagefilereader.cpp

```diff
@@ -39,8 +39,15 @@
 
 // OpenImageIO headers.
 #include "foundation/platform/_beginoiioheaders.h"
+#if __GNUC__ >= 7
```

Comment:
This is OpenImageIO headers. It seems that OIIO depends on OpenEXR and includes its headers, therefore we can place `#include "foundation/platform/_beginexrheaders.h` at the beginning of `_beginoiioheaders.h` and `#include "foundation/platform/_endexrheaders.h` at the end of `_endoiioheaders.h`.
So it will be beginEXRheaders beginOIIOheaders ... endOIIOheaders endEXRheaders